### ISSUE TITLE
BUG: Saved Queries not populating textarea

### ISSUE THREAD

**Issue by nogorilla** [beekeeper-studio/beekeeper-studio#1651]
**Describe the bug**
When loading a saved query, text area remains blank and does not show the query.

**To Reproduce**
1. Go to 'Save Queries'
2. Click on 'Any Saved Query'

**Expected behavior**
When a saved query is loaded, the content is displayed in the textarea and is editable

**Screenshots / Gifs**
<img width="2055" alt="Screenshot 2023-07-28 at 1 52 28 PM" src="https://github.com/beekeeper-studio/beekeeper-studio/assets/1786677/58b0a7fa-a0a6-49c4-bef8-e4f267236225">

**Version Information (please complete the following information):**
 - OS: osx 12.4.1 (c)
 - App Version: 3.9.18
 - Database type and version: CockroachDB v22.1.5

**Additional context**
When I open the Developer Tools > Console tab, the content of the query is displayed as a log line. Also, the query can still be run just not edited in any way

**Comment by klejejs** [comment]
I have the same issue with the latest release. Furthermore, I cannot input any text in the text field, even if a new query is created.

**Comment by not-night-but** [comment]
Hey! Sorry I totally missed this issue, I'll take a look today, but I suspect it may be a weird issue with the addition of vim mode. Any errors in the console?

**Comment by klejejs** [comment]
@not-night-but sorry for the dumb question, but how do I get to the console?
I downgraded to the previous version so that I can use the app, but I can upgrade it again to help you fix the issue.

**Comment by not-night-but** [comment]
Oh no it's not a dumb question, it's kinda hidden. In the menu bar, Help -> Show Developer Tools

**Comment by AndyBarker-egs** [comment]
Sounds like an issue discussed [here](https://github.com/beekeeper-studio/beekeeper-studio/issues/1640) (includes solution).

TLDR
I found that selecting the Editor type by clicking the Cog and selecting Default then restarting the app made it work. When I first selected the Cog, neither option was selected. I am guessing that the editor failed as it didn't know which editor to use!

**Comment by not-night-but** [comment]
Okay well that's actually good! I implemented a safeguard last week that should mean this behaviour isn't possible anymore. It will be in the next release. I'll leave this open until then to ensure it does fix the issue

**Comment by nogorilla** [comment]
just go `3.9.20` and can confirm the issue is resolved
